deployments/examples/gang/cmd/gangDeploy.sh

```diff
@@ -26,46 +26,22 @@ set -o pipefail
 JOBAMOUNT=$1
 GANGMEMBER=$2
 RUNTIMESEC=$3
+TIMEOUT=0
+
+# create service and job counter web server
+WORKDIR=$(cd "$(dirname "$0")"; pwd)
+kubectl apply -f $WORKDIR/../gang-coordinator.yaml
```

Comment:
please add a check here in case the `gang-coordinator.yaml` is not in the right place.
we can explicitly fail with an error: "gang-coordinator.yaml is not found in path xxxx"

Reply:
I had check several times, it can work in my local, do I miss something?

Reply:
I mean to check the file existence in the code, e.g

```
if [[ -f $WORKDIR/../gang-coordinator.yaml ]]; then
  // do something
else
  // print the error message and return
fi
```